packages/block-library/src/post-content/index.php

```diff
@@ -18,19 +18,21 @@ function render_block_core_post_content( $attributes, $content, $block ) {
 		return '';
 	}
 
-	if ( empty( get_the_content( $block->context['postId'] ) ) ) {
-		return '';
-	}
-
 	if ( ! in_the_loop() ) {
 		the_post();
 	}
 
+	$content = get_the_content( $block->context['postId'] );
```

Comment:
Shouldn't this be `get_the_content( null, false, $block->context['postId'] )` ?

```suggestion
	$content = get_the_content( null, false, $block->context['postId'] );
```